pkg/agent/cniserver/pod_configuration.go

```diff
@@ -198,6 +198,11 @@ func (pc *podConfigurator) configureInterfaces(
 	hostIface := result.Interfaces[0]
 	containerIface := result.Interfaces[1]
 
+	if !util.IsInfraContainer(containerNetNS) {
```

Comment:
It might look cleaner to split configureInterfaces() to two funcs, configureContainerLink(), and createContainerInterface() for OVS port and interface, and let the caller to decide which ones to call?
Or maybe pass a boolean argument of createOVSPort from the caller.

Reply:
Also seems line 214 to 224 should be removed?

Reply:
OK. I will pass a boolean argument in configureInterfaces to specify if createOVSPort is needed.

`line 214 to 224` should be remained because CRI may call CNIAdd multiple times on windows. These lines can avoid duplicate OVS port creation.

Done.